web-app/CKS.Web.Test/IntegrationTests/SearchTests.cs

```diff
@@ -11,19 +17,23 @@ public class SearchTests : TestBase
 		public SearchTests(ITestOutputHelper output) : base(output)
 		{ }
 
-		[Obsolete("Please replace this test with a real test when the search endpoint is setup")]
+
 		[Fact]
 		public async void ReturnsJson()
 		{
 			// Arrange
 			var client = _factory
-			   .CreateClient();
+				.WithFakeSearchProvider("FakeSearchResult-Cancer.json")
+				.CreateClient();
 
 			// Act
 			var response = await client.GetAsync("/api/search");
+			var responseBody = await response.Content.ReadAsStringAsync();
 
+			// Assert
 			response.StatusCode.ShouldBe(System.Net.HttpStatusCode.OK);
 			response.Content.Headers.ContentType.MediaType.ShouldBe("application/json");
+			responseBody.ShouldMatchApproved();
```

Comment:
This doesn't need to be an approvals test - can you load the FakeSearchResult-Cancer.json instead and assert that they're the same? We don't care what the content is, as long as it's the same as what search provider provided - that's the behaviour of the search endpoint